package.json

```diff
@@ -7,11 +7,9 @@
     "prepare:ganache": "mustache config/ganache.json subgraph.yaml.mustache > subgraph.yaml",
     "abi": "node src/scripts/fetchAbi",
     "addresses": "node src/scripts/printAddresses",
-    "compile": "truffle compile",
```

Comment:
This is good, in the past we were pulling and compiling the contracts, but then we changed the strategie. This compilation is not needed